src/pybind/mgr/alerts/module.py

```diff
@@ -200,15 +222,25 @@ def _send_alert_smtp(self,
                          diff: Dict[str, Any]) -> Optional[Dict[str, Any]]:
         # message
         self.log.debug('_send_alert_smtp')
+        # additional whitespace used to don't mess up formatting when disabled
+        emojis = {'HEALTH_OK': ' {}'.format(self.emoji_ok),
+                  'HEALTH_WARN': ' {}'.format(self.emoji_warn),
+                  'HEALTH_ERR': ' {}'.format(self.emoji_err)
+                 }
+        emote = ''
+        if self.emojis:
+            emote = emojis[status['status']]
```

Comment:
could use emojis.get(..., default_value) to handle the case where the status is not one of the expected ones.

Reply:
I must be honest... I din't know you could define a default value with get... Thx pushed